extensions/azurecore/src/azureResource/providers/databaseServer/databaseServerService.ts

```diff
@@ -3,26 +3,34 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { ServiceClientCredentials } from 'ms-rest';
-import { SqlManagementClient } from 'azure-arm-sql';
 
-import { azureResource } from '../../azure-resource';
-import { IAzureResourceService, AzureResourceDatabaseServer } from '../../interfaces';
+import { ResourceServiceBase, GraphData } from '../resourceTreeDataProviderBase';
+import { AzureResourceDatabaseServer } from '../../interfaces';
 
-export class AzureResourceDatabaseServerService implements IAzureResourceService<AzureResourceDatabaseServer> {
-	public async getResources(subscription: azureResource.AzureResourceSubscription, credential: ServiceClientCredentials): Promise<AzureResourceDatabaseServer[]> {
-		const databaseServers: AzureResourceDatabaseServer[] = [];
 
-		const sqlManagementClient = new SqlManagementClient(credential, subscription.id);
-		const svrs = await sqlManagementClient.servers.list();
+export interface DbServerGraphData extends GraphData {
+	properties: {
+		fullyQualifiedDomainName: string;
+		administratorLogin: string;
+	};
+}
 
-		svrs.forEach((svr) => databaseServers.push({
-			name: svr.name,
-			fullName: svr.fullyQualifiedDomainName,
-			loginName: svr.administratorLogin,
-			defaultDatabaseName: 'master'
-		}));
+export const serversQuery = 'where type == "microsoft.sql/servers"';
+
+export class AzureResourceDatabaseServerService extends ResourceServiceBase<DbServerGraphData, AzureResourceDatabaseServer> {
 
-		return databaseServers;
+	protected get query(): string {
+		return serversQuery;
+	}
+
+	protected convertResource(resource: DbServerGraphData): AzureResourceDatabaseServer {
+		// TODO: fullName should be updated to use IP/domain name + port
+		return {
```

Comment:
issue created for this?

Reply:
Apologies - was testing something else out and copy+paste left this here. Will remove.